localstack/utils/aws/dead_letter_queue.py

```diff
@@ -110,7 +113,7 @@ def _prepare_messages_to_dlq(source_arn: str, event: Dict, error) -> List[Dict]:
         messages.append(
             {
                 "Id": str(uuid.uuid4()),
-                "MessageBody": json.dumps(event),
+                "MessageBody": event,
```

Comment:
q: What if it's not a string but a dict? Was the `json.dumps(...)` here not necessary? Maybe it isn't necessary because it's json-encoded by the HTTP framework when sending the request anyways? 🤔

Reply:
When coming from SNS, it is always a string (either a RawMessage, or a JSON encoded message body).
It would double JSON encode the MessageBody (so, triple in total). If it's a dict, it will be JSON encoded only once like the rest of the message. It might be needed to check for the `application` protocol, as it still sends the `req_data`, but there's no reason the format isn't the same as all other notification, so I will fix it so that's always a string.